README.md

```diff
@@ -293,64 +289,104 @@ There are two ways to specify the role: use the built-in role or create a custom
     EOF
     ```
 
-    > available `AZURE_CLOUD_NAME` values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `AzureGermanCloud`
-
-### Option 2: Use AAD Pod Identity
+    > available `AZURE_CLOUD_NAME` values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`
 
-These instructions have been adapted from the [aad-pod-identity documentation][24].
+### Option 2: Use Azure AD Workload Identity
 
-Before proceeding, ensure that you have installed and configured [aad-pod-identity][23] for your cluster.
+These instructions have been adapted from the [Azure AD Workload Identity Quick Start][24] documentation.
 
-#### Create identity
+Before proceeding, ensure that you have installed [workload identity mutating admission webhook][28] and [enabled the OIDC Issuer][29] for your cluster.
 
 1. Create an identity for Velero:
 
     ```bash
-    export IDENTITY_NAME=velero
+    IDENTITY_NAME=velero
 
     az identity create \
         --subscription $AZURE_SUBSCRIPTION_ID \
         --resource-group $AZURE_RESOURCE_GROUP \
         --name $IDENTITY_NAME
 
-    export IDENTITY_CLIENT_ID="$(az identity show -g $AZURE_RESOURCE_GROUP -n $IDENTITY_NAME --subscription $AZURE_SUBSCRIPTION_ID --query clientId -otsv)"
-    export IDENTITY_RESOURCE_ID="$(az identity show -g $AZURE_RESOURCE_GROUP -n $IDENTITY_NAME --subscription $AZURE_SUBSCRIPTION_ID --query id -otsv)"    
+    IDENTITY_CLIENT_ID="$(az identity show -g $AZURE_RESOURCE_GROUP -n $IDENTITY_NAME --subscription $AZURE_SUBSCRIPTION_ID --query clientId -otsv)"
     ```
     
     If you'll be using Velero to backup multiple clusters with multiple blob containers, it may be desirable to create a unique identity name per cluster rather than the default `velero`.
 
-2. Assign the identity a role:
+2. Assign the identity roles:
 
     ```bash
-    export IDENTITY_ASSIGNMENT_ID="$(az role assignment create --role $AZURE_ROLE --assignee $IDENTITY_CLIENT_ID --scope /subscriptions/$AZURE_SUBSCRIPTION_ID --query id -otsv)"
+    az role assignment create --role $AZURE_ROLE --assignee $IDENTITY_CLIENT_ID --scope /subscriptions/$AZURE_SUBSCRIPTION_ID
     ```
 
-3. In the cluster, create an `AzureIdentity` and `AzureIdentityBinding`:
+    (Optional)Assign additional permissions to the service principal (For useAAD=true with built-in role)
 
+    If you use the custom role which has the blob data permissions, skip this step.
+
+    If you chose the AAD route, this is an additional permissions required for the identity to be able to access the storage account.
```

Comment:
nit: This is a little confusing, you are assigning a role to the client or Identity right?